src/frontend/src/icons/linear/index.tsx

```
@@ -0,0 +1,9 @@
+import React, { forwardRef } from "react";
+import LinearIconSVG from "./linear";
+
+export const LinearIcon = forwardRef<
+  SVGSVGElement,
+  React.PropsWithChildren<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)